resources/test/phantom/runner.js

```diff
@@ -0,0 +1,26 @@
+var page = require('webpage').create();
+var system = require('system');
+
+if (system.args.length !== 2) {
+  console.log('Expected a target URL parameter.');
+  phantom.exit(1);
+}
+
+page.onConsoleMessage = function (message) {
+  console.log(message);
+};
+
+var url = system.args[1];
+
+page.open(url, function (status) {
+  if (status !== "success") {
+    phantom.exit(1);
+  }
+
+  var exitCode = page.evaluate(function() {
```

Comment:
from the testing i did today it looks like phantom only returns a non zero status code when u explicitly do `phantom.exit(<non zero int>)`. I had to experiment with this a bit since I initially had tests that were failing but the CI wasn't showing as "failed"